pkg/activator/net/throttler.go

```diff
@@ -385,12 +388,15 @@ func (t *Throttler) activatorEndpointsUpdated(newObj interface{}) {
 	endpoints := newObj.(*corev1.Endpoints)
 
 	activatorCount := resources.ReadyAddressCount(endpoints)
+	t.logger.Debugf("Got %d ready activator endpoints.", activatorCount)
 	atomic.StoreInt32(&t.numActivators, int32(activatorCount))
 	t.updateAllThrottlerCapacity()
 }
 
 func (t *Throttler) activatorCount() int {
-	return int(atomic.LoadInt32(&t.numActivators))
+	ac := int(atomic.LoadInt32(&t.numActivators))
+	t.logger.Debugf("Returning %d activator endpoints.", ac)
```

Comment:
Given that we log activatorCount in both places we use it along with more context around the value do we still want this? Not a problem, but I try to avoid logging in getters due to the lack of context.

Reply:
Yeah, it ends up being a little noisy.  I dropped it.